From a636ce7ed4ec6248adf7618f70e5c3dc3b4a12c3 Mon Sep 17 00:00:00 2001
From: John Zhuge <jzhuge@apache.org>
Date: Sun, 28 May 2017 22:29:49 -0700
Subject: [PATCH 2547/2748] HADOOP-14464. hadoop-aws doc header warning #5
 line wrapped. Contributed by John Zhuge.

(cherry picked from commit 6c6a7a59622ba7c1e4faa5534f4479de0cd84993)
(cherry picked from commit f8a3de64f80e5a4922d6b541866b1cac0bbf9747)

Change-Id: I0a07d00c35f65d4575817b25df8a9f9049dc3a06
---
 .../src/site/markdown/tools/hadoop-aws/index.md    |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md b/hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md
index 31ff587..1fa9392 100644
--- a/hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md
+++ b/hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md
@@ -107,8 +107,7 @@ Do not inadvertently share these credentials through means such as
 
 If you do any of these: change your credentials immediately!
 
-### Warning #5: The S3 client provided by Amazon EMR are not from the Apache
-Software foundation, and are only supported by Amazon.
+### Warning #5: The S3 client provided by Amazon EMR are not from the Apache Software foundation, and are only supported by Amazon.
 
 Specifically: on Amazon EMR, s3a is not supported, and amazon recommend
 a different filesystem implementation. If you are using Amazon EMR, follow
-- 
1.7.9.5

